File: 프로그래머스/1/68935. 3진법 뒤집기/3진법 뒤집기.py
def solution(n):
    answer = 0
    tempList = []
    while (n):
        tempList.append(n % 3)
        n = n // 3
        
    tempList.reverse() 
    
    for i in range(len(tempList)):
        answer += tempList[i] * (3 ** i)
    
    return answer

File: 프로그래머스/1/68935. 3진법 뒤집기/3진법 뒤집기.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <cmath>

using namespace std;

vector<int> v;

int solution(int n) {
    int answer = 0;
    
    while (n) {
        v.push_back(n % 3);
        n = n / 3;
    }
    
    reverse(v.begin(), v.end());
    
    for (int i = 0; i < v.size(); i++) {
        answer += v[i] * pow(3, i);
    }
    
    return answer;
}